pkg/vm/engine/tae/model/colview.go

```diff
@@ -100,9 +100,10 @@ func (view *ColumnView) IsDeleted(row int) bool {
 }
 
 func (view *ColumnView) Close() {
-	if view.data != nil {
-		view.data.Close()
-	}
+	// view should never close anything.
```

Comment:
Here we should close the data if data != nil. The data may be allocated from the MPool if the block is in the memtable.